__test__/snapshot.js

```diff
@@ -1,5 +1,5 @@
 import { render } from '@testing-library/react';
-import Page from '../app/page';
+import Page from '../src/app/page';
```

Comment:
🟢 We can also use absolute path imports here for better organization and readability